### ISSUE TITLE
[BUG] Incorrect server response `/session/info`

### ISSUE THREAD

**Issue by sboh1214** [sparcs-kaist/otl-app#21]
## Description 
Same as title

## To Reproduce
Get response of `/session/info` and parse.

## Expected behavior
Client should get required property User.reviews[].userspecificIsLiked.

## Error Log
```
_TypeError (type 'Null' is not a subtype of type 'bool')
new Review.fromJson (/Users/sboh1214/Repos/otl-app/lib/models/review.dart:43)
new User.fromJson.<anonymous closure> (/Users/sboh1214/Repos/otl-app/lib/models/user.dart:77)
List.forEach (dart:core-patch/growable_array.dart:403)
new User.fromJson (/Users/sboh1214/Repos/otl-app/lib/models/user.dart:76)
InfoModel.getUser (/Users/sboh1214/Repos/otl-app/lib/providers/info_model.dart:67)
<asynchronous gap> (알 수 없는 소스:0)
InfoModel.getInfo (/Users/sboh1214/Repos/otl-app/lib/providers/info_model.dart:50)
<asynchronous gap> (알 수 없는 소스:0)
```

**Comment by sboh1214** [comment]
Fixed in sparcs-kaist/otlplus#816
by Add `"reviews": json_encode_list(profile.reviews.all()),` in apps/session/views.py `def info(request):`

**Comment by sboh1214** [comment]
Apply in sparcs-kaist/otlplus#826 
Prod Version [v3.2.3.0](https://github.com/sparcs-kaist/otlplus/releases/tag/v3.2.3.0)
